### DISCUSSION TITLE
Tip on how to load kubectx and kubens completions?

### DISCUSSION THREAD

**Discussion by AnimiVulpis** [marlonrichert/zsh-snap#154] [Q&A] [answered]
The helper scripts from https://github.com/ahmetb/kubectx are (still) bash scripts and I hoped that I could manage them by using `znap`.

I tried to put

```
znap source ahmetb/kubectx kubectx kubens
```

into my `~/.zshrc` file, but when I do that and run `znap restart` I get

```
Validating dotfiles...
Validation failed with the following errors:
/somefolder/zsh-repos/kubectx/kubectx:set:21: no such option: u
/somefolder/zsh-repos/kubectx/kubens:set:21: no such option: u
Restart aborted.
```

For completeness:
`zstyle ':znap:*' repos-dir ~/zsh-repos` is how `znap` is configured (`~/zsh-repos` being symlinked to `/somefolder/zsh-repos`)

---
**Bonus**

I also wanted to do

```
znap source ahmetb/kubectx kubectx kubens completion/_kubectx.zsh completion/_kubens.zsh
```

to have completions too. But as the first failed I didn't check further.

Any ideas?

**Comment by marlonrichert** [comment, marked as answer]
`ahmetb/kubectx` is _not_ a Zsh plugin. So, `znap source ahmetb/kubectx` will _not_ work.

Instead, run this once on the command line:
```zsh
znap install ahmetb/kubectx
```

Then restart your shell.

**Reply by AnimiVulpis** [reply]
Thank you very much. That is awesome. Keep up the great work

**Reply by marlonrichert** [reply]
You're welcome! And if you enjoy using my software, please consider [sponsoring me](https://github.com/sponsors/marlonrichert). 🙂
